### ISSUE TITLE
Change schema of schema api with latest mapping osm

### ISSUE THREAD

**Issue by meomancer** [healthsites/healthsites#1128]
depend on #1129
# Problem
We already has finished mapping, but it is not implemented on our schema. 
And we need a system that schema can generated automatically

# Solution
- Create a system that automatically generate schema based on this validator : https://github.com/healthsites/healthsites/issues/1129
- Or we can do in reverse, validator gets the validation from this schema
- Update https://github.com/healthsites/healthsites/blob/master/django_project/api/schema.json

**Comment by meomancer** [comment]
Already done by @myarjunar
